Source/WebCore/animation/CSSPropertyAnimation.cpp

```diff
@@ -1023,13 +1023,13 @@ class LengthPointOrAutoPropertyWrapper : public LengthPointPropertyWrapper {
 
 private:
     // Check if it's possible to interpolate between the from and to values. In particular,
-    // it's only possible if they're both not auto.
+    // it's only possible if they're both not auto or normal.
     bool canInterpolate(const RenderStyle& from, const RenderStyle& to, CompositeOperation) const final
     {
-        bool fromIsAuto = value(from).x().isAuto() && value(from).y().isAuto();
-        bool toIsAuto = value(to).x().isAuto() && value(to).y().isAuto();
+        auto valueFrom = value(from);
+        auto valueTo = value(to);
 
-        return (!fromIsAuto && !toIsAuto); 
+        return (!valueFrom.x().isAuto() && !valueTo.x().isAuto() && !valueFrom.x().isNormal() && !valueTo.x().isNormal());
```

Comment:
nit: you can remove the brackets around the return value

Reply:
Fixed.